tools/strip_nibabel

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Convert NiBabel objects in PyMVPA datasets attributes into a portable form
```

Comment:
I considered it a one time helper. Current PyMVPA can ready old datasets transparently anyway, and write new-style ones automatically.